rollup/rollup.config.js

```diff
@@ -0,0 +1,75 @@
+// rollup.config.js
+import sass from 'node-sass'
+import postcss from 'rollup-plugin-postcss'
+import postcssModules from 'postcss-modules'
+import autoprefixer from 'autoprefixer'
+import CssModulesSassLoader from './css-modules-sass-loader'
+
+import commonjs from 'rollup-plugin-commonjs';
+import typescript from 'rollup-plugin-typescript2';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import dot from 'rollup-plugin-dot';
+
+const cssExportMap = {};
+
+const sassPreprocessor = (content, id) => new Promise((resolve) => {
+  const result = sass.renderSync({ file: id });
+
+  resolve({ code: result.css.toString() });
+});
+
+export default {
+  input: 'src/index.ts',
+  plugins: [
+    nodeResolve({
+      jsnext: true,
+      main: true
+    }),
+    commonjs({
+      include: 'node_modules/**',  // Default: undefined
+      extensions: [ '.js', '.ts' ],  // Default: [ '.js' ]
+
+      ignoreGlobal: false,  // Default: false
+
+      sourceMap: false,  // Default: true
+
+      namedExports: {
+        'node_modules/eventemitter3/index.js': [ 'EventEmitter' ],
+        //'node_modules/css-element-queries/index.js': [ 'ElementQueries' ],
+      }
+    }),
+    postcss({
+      extract: false,//path.resolve('./dist/components.css'),
+      sourceMap: false,
+      extensions: ['.scss'],
+      preprocessor: sassPreprocessor, // Pre-process all imports with Sass
+      plugins: [
+        autoprefixer(),
+        postcssModules({
+          Loader: CssModulesSassLoader, // Load all "composes" files with Sass
+          generateScopedName: '[name]__[local]___[hash:base64:5]',
+          getJSON(id, exportTokens) {
+            cssExportMap[id] = exportTokens
+          }
+        })
+      ],
+      getExportNamed: false,
+      getExport(id) {
+        return cssExportMap[id]
+      }
+    }),
+    dot({
+      include: ['**/*.dot', '**/*.tpl'], // default to '**/*.dot'
+			exclude: ['**/index.html']
+    }),
+    typescript(),
+    uglify(),
+  ],
+  exports: 'named',
+  name: 'VideoPlayer',
+  output: {
+    file: 'dist/player.js',
```

Comment:
`dist/playable.X.min.js`?
Seems we need `min` and NOT `mind` versions. 

Mb use `rollup.config.js` and `rollup.min.config.js`.
Or just remove `uglify` plugin from rollup and run `uglify` on `rollup` output.

Smth like:
```sh
> rollup --config rollup/rollup.config.js src/index.ts dist/playable.rollup.js   | uglifyjs  --compress --mangle > dist/playable.rollup.min.js
```